product_docs/docs/pem/8/pem_online_help/05_toc_pem_management_basics/10_managing_cluster_objects/06_role_reassign_dialog.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Role Reassign/Drop Own Dialog"
+---
+
+<div id="role_reassign_dialog" class="registered_link"></div>
+
+Use the *Reassign/Drop Own* dialog to change the ownership of database objects owned by a database role. This dialog instructs the system to change the ownership of database objects owned by any of the *old_roles* to *new_role*.
+
+The *Reassign/Drop Own* dialog organizes the Reassign & Drop role through General tab.
+
+<img src="../../images/role_reassign_dialog.png" class="align-center" alt="Reassign/Drop Own dialog" />
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)